fdbserver/TLogGroup.actor.cpp

```diff
@@ -0,0 +1,293 @@
+/*
+ * TLogGroup.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbclient/CommitProxyInterface.h"
+#include "fdbclient/FDBTypes.h"
+#include "fdbclient/SystemData.h"
+#include "fdbrpc/Replication.h"
+#include "fdbrpc/ReplicationPolicy.h"
+#include "fdbserver/TLogGroup.h"
+#include "fdbserver/WorkerInterface.actor.h"
+#include "flow/Arena.h"
+#include "flow/FastRef.h"
+#include "flow/flow.h"
+#include "flow/Error.h"
+#include "flow/UnitTest.h"
+#include "flow/serialize.h"
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <unordered_map>
+#include <vector>
```

Comment:
The https://google.github.io/styleguide/cppguide.html style guide prefers order these before FDB's headers, separated by a blank line.